Sources/MongoKitten/CollectionHelpers/Collection+Indexes.swift

```diff
@@ -24,6 +24,30 @@ extension MongoCollection {
         }._mongoHop(to: hoppedEventLoop)
     }
     
+    /// Create 1 or more indexes on the collection.
+    /// - Parameter indexes: A collection of indexes to be created.
+    /// - Returns: A future indicating success or failure.
+    public func createIndexs(_ indexes: [CreateIndexes.Index]) -> EventLoopFuture<Void> {
+        guard transaction == nil else {
```

Comment:
Can the above implementation now call this one, as to not duplicate code?

Reply:
Done ✅